kedro/config/logging.yml

```diff
@@ -33,15 +33,21 @@ handlers:
 
 loggers:
     anyconfig:
-        level: WARNING
-        handlers: [console]
+        level: INFO
```

Comment:
Why `anyconfig` level is changed from `WARNING` to `INFO`?

Reply:
Oops - thanks for the catch!